docs/Windows-and-macOS-Installation.md

```diff
@@ -6,8 +6,8 @@ To support the necessary Bluetooth APIs leveraged within NXBT, installation with
 
 Before continuing, please ensure you have the following:
 
-- A **USB** Bluetooth Adapter
-    - Internal Bluetooth adapters are incompatible (generally) with the process that allows a VM to use external resources.
+- A **USB** Bluetooth Adapter (for macOS)
+    - For internal Bluetooth adapters, Windows machines (generally) support disabling Bluetooth for themselves and allowing a VM to capture it, but macOS machines generally don't allow their own Bluetooth to be disabled and captured.
```

Comment:
[This stackexchange post](https://apple.stackexchange.com/questions/391225/getting-bluetooth-on-windows-10-virtual-machine-guest-virtualbox-disabling) combined with this [VirtualBox ticket](https://www.virtualbox.org/ticket/2372) from 10 years ago appears to explain why this wouldn't work on macOS (or would require many more difficult workarounds).